docs/access_token-pop.md

```diff
@@ -0,0 +1,178 @@
+
+   2.2	The AS binds (sender-constrains) the access token to the public key claimed by the client; that is, the access token cannot be used without proving possession of the respective private key. This is signaled to the client by using the token_type value "pop" and by appending a "cnf" claim in the access token jwt containing a "kid" member identifying the public key.
+
+   2.3	If the client wants to use the access token, it has to prove possession of the private key by adding a header to the request that, again, contains a JWT signed with this private key (Signed Http Request). The JWT contains the endpoint URL and the request method. The resource server needs to receive information about which public key to check against. This information is either encoded directly into the access token, for JWT structured access tokens, or provided at the token introspection endpoint of the authorization server (request not shown).
+
+   2.4	 The resource server refuses to serve the request if the signature check fails or the data in the JWT do not match, e.g., the request URI does not match the URI claim in the JWT.
+
+3. **How does Client -> Resource Provider protocol change between Bearer and Pop access tokens**.
+
+   In the bearer flow, when a client requests a resource from an RP, the client provides an **authorization header** containing the AT. 
+
+   In the PoP flow, when a client requests a resource from an RP, the client provides an authorization header containing the Signed Http Request**(SHR)**, which itself contains the AT: 
+
+   |                               | **Bearer**            | **PoP**                |
+   | ----------------------------- | --------------------- | ---------------------- |
+   | **Authentication** **Method** | Bearer Authentication | PoP Authentication     |
+   | **AT issued by STS**          | “Bearer” AT           | “PoP” AT               |
+   | **Authorization Header**      | Bearer AT             | **SHR** **{ PoP AT }** |
+
+4. **Configure MSAL to request Proof-of-Possession Access tokens**
+
+   4.1	For an interactive acquireToken request, create an instance of MSALInteractiveTokenParameters as shown below.
+
+   **MSALInteractiveTokenParameters**
+
+   ```objective-c
+   #if TARGET_OS_IPHONE
+       UIViewController *viewController = ...; // Pass a reference to the view controller that should be used when getting a token interactively
+       MSALWebviewParameters *webParameters = [[MSALWebviewParameters alloc] initWithAuthPresentationViewController:viewController];
+   #else
+       MSALWebviewParameters *webParameters = [MSALWebviewParameters new];
+   #endif 
+   
+   MSALInteractiveTokenParameters *interactiveParams = [[MSALInteractiveTokenParameters alloc] initWithScopes:scopes webviewParameters:webParameters];
```

Comment:
Do you need to set the auth scheme POP?

Reply:
It is explained below in step 5.3 of the doc.